src/org/thoughtcrime/securesms/ProfileGalleryFragment.java

```diff
@@ -184,39 +188,84 @@ public void onMediaLongClicked(DcMsg mediaRecord) {
     }
   }
 
-  @SuppressLint("StaticFieldLeak")
-  private void handleDeleteMedia(@NonNull Collection<DcMsg> mediaRecords) {
-    int recordCount       = mediaRecords.size();
-    Resources res         = getContext().getResources();
-    String confirmMessage = res.getQuantityString(R.plurals.ask_delete_messages,
-                                                  recordCount,
-                                                  recordCount);
-
-    AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
-    builder.setMessage(confirmMessage);
-    builder.setCancelable(true);
-
-    builder.setPositiveButton(R.string.delete, (dialogInterface, i) -> {
-      new ProgressDialogAsyncTask<DcMsg, Void, Void>(getContext(),
-                                                                         R.string.one_moment,
-                                                                         R.string.one_moment)
-      {
-        @Override
-        protected Void doInBackground(DcMsg... records) {
-          if (records == null || records.length == 0) {
-            return null;
-          }
-
-          for (DcMsg record : records) {
-            dcContext.deleteMsgs(new int[]{record.getId()});
-          }
-          return null;
-        }
-
-      }.execute(mediaRecords.toArray(new DcMsg[mediaRecords.size()]));
+  private void handleDisplayDetails(DcMsg dcMsg) {
+    String info_str = dcContext.getMsgInfo(dcMsg.getId());
```

Comment:
minor style thing: I think we're mostly using camelCase in DC Android

Reply:
that was copy+pasted from somewhere, makes sense to change also in ConversationFragment.java